Library/Homebrew/sorbet/tapioca/compilers/args.rb

```diff
@@ -2,10 +2,17 @@
 # frozen_string_literal: true
 
 require_relative "../../../global"
+require "cli/parser"
 
 module Tapioca
   module Compilers
     class Args < Tapioca::Dsl::Compiler
+      GLOBAL_OPTIONS = T.let(
+        Homebrew::CLI::Parser.global_options.map { _1.slice(0, 2) }.flatten
+             .map { "#{Homebrew::CLI::Parser.option_to_name(_1)}?" }.freeze,
```

Comment:
```suggestion
        Homebrew::CLI::Parser.global_options.each_with_object([]) do |(short_option, long_option), array|
          array << "#{Homebrew::CLI::Parser.option_to_name(short_option)}?"
          array << "#{Homebrew::CLI::Parser.option_to_name(long_option)}?"
        end.freeze,
```

Nit: This is a bit opaque. It's not obvious without looking at `.global_options` what the format could be for the block arguments. I don't have any suggestions that are equally succinct though. I have not tested the above code locally.

Reply:
Great suggestion, PTAL at the next commit

Reply:
Much more readable, thanks!